TRAIN.md

```diff
@@ -263,23 +263,21 @@ You can find examples configuration files in [`configs/oc22/s2ef`](https://githu
 
 ## Joint Training
 
-Training on OC20 total energies whether independently or jointly with OC22 requires `total_energy: True` and a path to the `oc20_ref` (download link provided below) to be specified in the configuration file. These are necessary to convert OC20 adsorption energies into their corresponding total energies. The following changes in the configuration file capture these changes:
+Training on OC20 total energies whether independently or jointly with OC22 requires a path to the `oc20_ref` (download link provided below) to be specified in the configuration file. These are necessary to convert OC20 adsorption energies into their corresponding total energies. The following changes in the configuration file capture these changes:
```

Comment:
I'd keep the `total_energy: True|False` flag. It's much more explicit in conveying whether to train on total energies or not.

Reply:
I added a `train_on_oc20_total_energies` flag.